src/mmrelay/cli.py

```
@@ -1506,11 +1743,324 @@ def handle_auth_command(args: argparse.Namespace) -> int:
+                    cursor = conn.execute("PRAGMA journal_mode;")
+                    mode = cursor.fetchone()[0]
+                    if mode.lower() == "wal":
+                        print("      Journal: WAL mode ✅")
+                    else:
+                        print(f"      Journal: {mode} mode")
+            except (sqlite3.Error, OSError):
+                pass  # Don't fail if we can't check
+        else:
+            print(f"   ⚠️  Database not found: {db_path}")
+            print("       Will be created on first run")
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)